testhelpers/testflags/flags.go

```diff
@@ -0,0 +1,64 @@
+package testflags
+
+import (
+	"flag"
+	"testing"
+)
+
+// Test enablement flags
+var functionalTest = flag.Bool("functional", false, "Run the functional go tests")
+var integrationTest = flag.Bool("integration", false, "Run the integration go tests")
+var unitTest = flag.Bool("unit", false, "Run the unit go tests")
+
+// Test modifiers
+// concurrent flag will cause tests to be ran in parallel, its default value is `true`.
+var concurrent = flag.Bool("concurrent", true, "Runs the go test in parallel")
```

Comment:
The built in `-parallel=1` flag can be used to disable concurrency, so I don't think this is needed. https://golang.org/cmd/go/#hdr-Testing_flags

Reply:
Good point, I'll remove it.